vault/src/execution.rs

```diff
@@ -476,13 +476,9 @@ pub async fn execute_open_requests(
             tokio::task::yield_now().await;
         }
 
-        let tx = match result {
-            Ok(x) => x,
-            Err(e) => {
-                tracing::warn!("Failed to process transaction: {}", e);
-                continue;
-            }
-        };
+        // Bail on any errors because if we miss any transactions we could end up
+        // double paying
+        let tx = result?;
```

Comment:
I thought we did not want to did this because this there may be un-processable transactions which would prevent the Vault from continuing to make valid payments?

Reply:
We either have to risk not paying, or to risk double paying. Of those 2, I'd argue that double paying is worse